packages/astro/package.json

```diff
@@ -66,7 +69,6 @@
       "default": "./zod.mjs"
     },
     "./middleware": {
-      "types": "./dist/core/middleware/index.d.ts",
```

Comment:
We still need to keep this to support node16 and above support. The `typesVersion` trick only exists to support versions lower than that.

Reply:
Hmm... I thought it wouldn't be a problem.

I added it back :)